tests/vmi_multus_test.go

```diff
@@ -875,6 +872,57 @@ var _ = Describe("SRIOV", func() {
 		It("[test_id:3957]should connect to another machine with sriov interface over IPv6", func() {
 			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
+
+		Context("VLANed SR-IOV network", func() {
+			var (
+				vlanedVMI1   *v1.VirtualMachineInstance
+				vlanedVMI2   *v1.VirtualMachineInstance
+				nonVlanedVMI *v1.VirtualMachineInstance
+			)
+
+			const (
+				cidrVlaned1            = "192.168.0.1/24"
+				cidrVlaned2            = "192.168.0.2/24"
+				cidrNonVlaned          = "192.168.0.3/24"
+				sriovVlanNetworkName   = "sriov-vlan"
+				sriovNoVlanNetworkName = "sriov-no-vlan"
+			)
+
+			BeforeEach(func() {
+				createNetwork(sriovVlanNetworkName, tests.NamespaceTestDefault, sriovConfVlanCRD)
+				createNetwork(sriovNoVlanNetworkName, tests.NamespaceTestDefault, sriovConfNoVlanCRD)
+				vlanedVMI1 = createSriovVMI([]string{sriovVlanNetworkName})
+				// manually configure IP/link on sriov interfaces because there is
+				// no DHCP server to serve the address to the guest
+				configInterface(vlanedVMI1, "eth1", cidrVlaned1, "\\# ")
+			})
+
+			AfterEach(func() {
+				Expect(deleteNetwork(sriovVlanNetworkName, tests.NamespaceTestDefault)).To(Succeed())
```

Comment:
Could you please add Expect error messages?

Reply:
can you elaborate? what info do you think is missing?

Reply:
https://github.com/kubevirt/kubevirt/pull/3479/files#r468468287

Reply:
same as https://github.com/kubevirt/kubevirt/pull/3479#discussion_r468487765